src/__test__/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.test.jsx

```diff
@@ -67,42 +67,51 @@ const customAPIResponses = {
   [`/plots-tables/${plotUuid}`]: () => statusResponse(404, 'Not Found'),
 };
 
-const defaultMockResponses = _.merge(
+const mockAPIResponse = _.merge(
   generateDefaultMockAPIResponses(experimentId),
   customAPIResponses,
-  mockWorkerResponses,
 );
 
 const defaultProps = { experimentId };
 const dotPlotPageFactory = createTestComponentFactory(DotPlotPage, defaultProps);
```

Comment:
I had a peek at the `test-utils` folder and it seems to be going towards the "utils" direction, meaning that there is no real structure there and soon it will be messy. If possible, I suggest to introduce some order there and organise files there into different folders

Reply:
it doesn't have to be as part of this PR, but it seems to me that it needs to be done soon